sdk/schemaregistry/Microsoft.Azure.Data.SchemaRegistry.ApacheAvro/src/SchemaRegistryAvroEncoder.cs

```diff
@@ -58,19 +54,114 @@ private enum SupportedType
             GenericRecord
         }
 
-        private static SupportedType GetSupportedTypeOrThrow(Type type)
+        #region Encode
+
+        /// <summary>
+        /// Encodes the message data into Avro and stores it in <see cref="MessageWithMetadata.Data"/>. The <see cref="MessageWithMetadata.ContentType"/>
```

Comment:
nit:  This reads a bit awkward to me.   Maybe we should consider:

```suggestion
        /// Encodes the message data as Avro and stores it in <see cref="MessageWithMetadata.Data"/>. The <see cref="MessageWithMetadata.ContentType"/>
```

Reply:
alternatively, this would also work:

```suggestion
        /// Encodes the message data into Avro format and stores it in <see cref="MessageWithMetadata.Data"/>. The <see cref="MessageWithMetadata.ContentType"/>
```